### ISSUE TITLE
Gregicality multiblocks?

### ISSUE THREAD

**Issue by KaktuszSok** [GregTech-Intergalactical/GregTech#100]
Hi, I'm wondering if you have plans to implement multiblocks from Gregicality in the future. I am mainly thinking of the pyrolyse oven, large generators and large macerator/assembly line/other machines, but they also have a lot of unique endgame content. Using multiblocks adds a lot to the experience and gives good reason to build more realistically sized factories, powerplants etc.
Thanks!

**Comment by Vliro** [comment]
Yes. Not sure exactly which or if is somewhat different but definitely some.
